pkg/v1/tkg/test/framework/schemes.go

```diff
@@ -17,6 +17,7 @@ import (
 	controlplanev1 "sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1beta1"
 	expv1 "sigs.k8s.io/cluster-api/exp/api/v1beta1"
 
+	kapppkgv1alpha1 "github.com/vmware-tanzu/carvel-kapp-controller/pkg/apiserver/apis/datapackaging/v1alpha1"
```

Comment:
nit: Can we change this something like kappcontrollerv1alpha1 or something. `kappppkgv1alpha1` doesn't read that well in code IMO. Please let me know if think otherwise

Reply:
updated.